cloudformation/proxy/proxy.yaml

```diff
@@ -120,6 +120,11 @@ Resources:
         - CidrIp: !GetAtt PrivateSubnet.CidrBlock
           Description: Allow all inbound traffic from private subnet
           IpProtocol: -1
+        - CidrIp: 0.0.0.0/0
```

Comment:
This rule will generate a high severity security violation. 
To prevent this i suggest to do as follows:
1. add an optional parameter for the CIDR to allow SSH traffic from (with default value: 0.0.0.0/0)
2. from the test you can inject the CIDR needed by the test

Reply:
Nice catch! I noticed that, this is just for drafting PR and locally test. Will fix it.

Reply:
Done!